.envrc

```diff
@@ -0,0 +1,28 @@
+layout node
+
+# You can define your system-specific logic (like Git settings or GH tokens) in .envrc.local
+# If that logic is usable by other people and might improve development environment, consider
+# contributing it to this file ♥
+
+watch_file .envrc.local
+[ -f .envrc.local ] && {
+    source .envrc.local    
+}
+
+if [ -z "${DONTNIX}" ] && has nix; then
+
+      if has lorri && has use_lorri; then
```

Comment:
i don't think `use_lorri` is a thing because i had to define it myself. i also think checking it first might not be a good idea, people might prefer using flakes (i do!)

how about we drop the lorri part here and just support the standard `use flake` and `use nix`? someone who wants lorri can still use it in their .envrc.local.